Library/Formula/riak.rb

```diff
@@ -1,8 +1,6 @@
-require "formula"
-
 class Riak < Formula
-  desc "Distributed database"
   homepage "http://basho.com/riak/"
+  desc "Riak is a distributed NoSQL key-value data store."
```

Comment:
This line should be before `homepage` and not start with the formula name (`Riak is a distributed...` → `Distributed…`).